test/GeoCoding/SarIndex2CoordinatesTest.jl

```diff
@@ -0,0 +1,81 @@
+
+
+function approx_point_test() 
+    ## Arrange
+    # data from metadata file geolocationGridPoint (10598, 6055)
+    
+    theta = 3.784327810329249e+01 *pi/180 
+    expected_point =  [3.858889230794824e+01, 
+        -2.706292464272735e+01,
+         1.234132796525955e-04]
+
+    orbit_states = SARProcessing.load_precise_orbit_sentinel1(PRECISE_ORBIT_TEST_FILE2)
+    image = load_test_slc_image()
+    interpolator = SARProcessing.orbit_state_interpolator(orbit_states,image.metadata)
+    time =  SARProcessing.parse_delta_time("2022-09-18T07:49:40.819491",image.metadata.reference_time)
+
+    orbit_state = interpolator(time)
+
+    ## act 
+    point = SARProcessing.approx_point(orbit_state,theta)
+
+    point_geodetic = SARProcessing.ecef2geodetic(point)
+    
+    #Assert
+    test_ok = isapprox(point_geodetic[1] * 180/pi, expected_point[1], atol=1.0)
+    test_ok &= isapprox(point_geodetic[2] * 180/pi, expected_point[2], atol=1.0)
+    test_ok &= isapprox(point_geodetic[3], 0.0, atol=1.0)
```

Comment:
why not expected_point[3]?

Reply:
The "approx_point" function finds the intersection between the line of sight unit vector and the ellipsoid. The height should thus be 0